frontend/views/containers/payments/PaymentRowRecord.vue

```diff
@@ -8,7 +8,7 @@
           i18n.sr-only Mark as sent
 
     template(slot='cellActions')
-      .cpr-date.pill.is-danger(v-if='payment.isLate' data-test='isLate') {{ humanDate(payment.date) }}
```

Comment:
This appears to be a mistake... as the [updated designs](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=23189%3A249864) still insist that this is only displayed on late payments.

Reply:
@taoeffect yes, I think I temporarily removed it for testing and forgot to put it back. let me add it back.